packages/app/components/Queue/QueueListSharedComponents.tsx

```diff
@@ -105,6 +111,12 @@ export function QueueInfoColumn({
           <QueuePropertyText>{queue.notes}</QueuePropertyText>
         </QueuePropertyRow>
       )}
+      <QueuePropertyRow>
+        {isQueueLive ? <CloudSyncOutlined /> : <FrownOutlined />}
```

Comment:
@liustanley we await a better design

Reply:
YEAH FROWN OUTLINED I'M LIVING FOR THIS

Reply:
i fixed this in my branch rip but its all good i'm here for frown outlined

Reply:
oh wait just kidding this is something new i think it looks good!